### PR TITLE
Removed package name property capabilities + commented capabilities from cl…

### PR DESCRIPTION
**Pull request by Jordan-Gilliam**
…assName. These properties are interfering with our naming structure. Id be happy to document the changes as optional capabilities in the readme.

### CODE REVIEW

File: lib/reporter.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ class JunitReporter extends events.EventEmitter {
             ? this.options.suiteNameFormat
             : /[^a-zA-Z0-9]+/
 
-        this.on('end', ::this.onEnd)
+        this.on('end', this.onEnd)
```
Comment (BorisOsipov) [inline lib/reporter.js]: is it ok or misprint?

File: lib/reporter.js [hunk 2]
```diff
@@ -109,15 +109,15 @@ class JunitReporter extends events.EventEmitter {
                         .time(suite.duration / 1000)
                         .property('specId', specId)
                         .property('suiteName', suite.title)
-                        .property('capabilities', capabilities.sanitizedCapabilities)
+                        // .property('capabilities', capabilities.sanitizedCapabilities)
```
Comment (BorisOsipov) [inline lib/reporter.js]: why?

File: lib/reporter.js [hunk 3]
```diff
@@ -109,15 +109,15 @@ class JunitReporter extends events.EventEmitter {
...
                         .property('file', spec.files[0].replace(process.cwd(), '.'))
 
                     for (let testKey of Object.keys(suite.tests)) {
                         if (testKey !== 'undefined') { // fix cucumber hooks crashing reporter
                             const test = suite.tests[testKey]
                             const testName = this.prepareName(test.title)
                             const testCase = testSuite.testCase()
-                                .className(`${packageName}.${suiteName}`)
+                                .className(`${suiteName}`)
```
Comment (BorisOsipov) [inline lib/reporter.js]: why?

**Comment by BorisOsipov** [comment]
Hi @Jordan-Gilliam, i don't understand why we need such changes. Could you please describe more.

File: lib/reporter.js (hunk 2 above)
Comment (Jordan-Gilliam) [reply lib/reporter.js]: This should be an option not a default. Ideally set in the config.

File: lib/reporter.js (hunk 3 above)
Comment (Jordan-Gilliam) [reply lib/reporter.js]: This should also be an option, not a default.

File: lib/reporter.js (hunk 1 above)
Comment (Jordan-Gilliam) [reply lib/reporter.js]: I think this was a misprint.

File: lib/reporter.js (hunk 2 above)
Comment (BorisOsipov) [reply lib/reporter.js]: ok make options to control it.

File: lib/reporter.js (hunk 3 above)
Comment (BorisOsipov) [reply lib/reporter.js]: the same.

**Comment by Jordan-Gilliam** [comment]
Fixing issue I will resubmit once this is fixed. Thank you for the discussion.

**Pull request closed without merging**